presentations/en/SLF.md

```diff
@@ -0,0 +1,77 @@
+
+Quote of Mark Schwartz
+> To work effectively in the digital world, you must first accept complexity and uncertainty, for they demand very different approach to carrying out initiatives. A predictable world rewards advanced planning and rigid plan execution. But a complex and uncertain world rewards an empirical cycle of trying, observing, and correcting.
+
+Talk about 4 angles the team is addressing using strategies (mandate)
+
+- Diagnose and Recommend changes on problems that inhibit the desired changes in behaviours
+  - example: Heavy Governance promotes risks, need to reduce burden of giving permission to work
+  - example: Senior Advisors release process mapping involving 13 different teams and up to 49 email exchanges, need for embedded teams
+- Measuring strategy progress using non-invasive means and making course correction from emperical evidence
+- Challenge current thinking and approaches
+  - example: No more status update meetings due to modern tools (KANBAN boards in the open)
+  - example: This presentation not working in IE and refusal to make it work in IE because we're focusing on modern tech
+  - example: Technical work needing to be reviewed by non technical staff for approval
+  - example: Defaults over standards (if an expert in their respective area proposes a different approach than the departmental 'default' they are free to do so -- perhaps that will become the next default)
+
+--
+
+### 2. Focused Areas
+
+- Moving to Product Management
+  - Raising awareness on inadequate funding model for the Digital World
+  - Moving to embedded and dedicated product teams
+  - Ability to deploy and manage production workloads
+  - Lessons learned from failed attempts (e.g. PSPC)
+- 21 DevOps capabilities (DORA), including job satisfaction
+- Adoption of Open Source, as a philosophy
+- Empowering employees
+- Automation to allow staff to focus on meaningful work
+
+Link to [Strategy Map](https://sara-sabr.github.io/ITStrategy/strategy-summary.html)
+
+--
+
+### 3. Recommended immediate actions
+
+1. Continuous Learning (20% dedicated to learn, automate, and improve)
+2. Make ongoing work and backlog visible, including code
+3. Allow onboarding of modern tools, especially open source ones
+4. Avoid toil, invest in automation
+5. Promote smaller investments over large ones by leveraging Lite Projects
+
+Link to [Strategies and Actions](https://sara-sabr.github.io/ITStrategy/strategies-actions.html)
```

Comment:
We should introduce our Quarterly strategies (actions) strategy.  Present the first one that will start Q1, 20% time continuous improvement. Talk about the [Goals](https://github.com/sara-sabr/ITStrategy/blob/master/_pages/en/strategy-learning-automating-improving.md#goals) and [IITB teams must (actions)](https://github.com/sara-sabr/ITStrategy/blob/master/_pages/en/strategy-learning-automating-improving.md#iitb-teams-must).

Reply:
I added a blurb about this here 

https://github.com/sara-sabr/ITStrategy/pull/515/commits/62beee1d5a3a5dfdef935bb4369d17b1430d44df